README.md

```diff
@@ -1,142 +1,220 @@
 
-WSO2 EI is a unified distribution of WSO2 Enterprise Service Bus and Data Services Server, which works as a single runtime (Integrator) along with optional runtimes for WSO2 Analytics, Business Processor and Message Broker. This product distribution includes a shared component directory, with profile-based management capabilities for each runtime.
-
-WSO2 EI simplifies integration by allowing users to easily configure message routing, inter-mediation,transformation, logging, task scheduling, load balancing, failover routing, event brokering etc. Data services and various applications can also be hosted and exposed using WSO2 EI. These integration capabilities are further powered by the capabilities of the WSO2 Analytics, Business Processor and Message Broker runtimes.
-
-## Installation & Running
-
-## Running the Integrator
-
-1. Extract  wso2ei-6.5.0.zip and go to the extracted directory/bin.
-2. Run integrator.sh or integrator.bat.
-3. Point your favourite browser to  <https://localhost:9443/carbon>
-4. Use the following username and password to login
-   username : admin
-   password : admin
-
-## Running other runtimes individually (Analytics, Broker, Business-Process)
-
-1. Extract wso2ei-6.5.0 and go to the extracted directory.
-2. Go to wso2ei-6.5.0/wso2 directory.
-3. Go to appropriate runtime directory (analytics/broker/business-process) /bin.
-4. Execute wso2server.sh or wso2server.bat.
-5. Access the url related to the required runtime. (For example, use <https://localhost:9445/carbon> for the business-process runtime.)
-
-## Known Issues
-
-All the open issues pertaining to WSO2 Enterprise Integrator are reported at the following location:
+![WSO2 Enterprise Integrator](gh-docs/images/wso2-integration-logo.png?raw=true)
+
+[![Build Status][badge-travis-image]][badge-travis-url]
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/wso2/product-ei/blob/master/LICENSE)
+[![Twitter](https://img.shields.io/twitter/follow/wso2.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=wso2)
+
+# WSO2 Enterprise Integrator
+
+WSO2 Enterprise Integrator is an open source, fast, cloud native and scalable integration solution which can be 
+positioned as the core of WSO2 Integration Agile Platform. It enables enterprise integration developers to build 
+sophisticated integration solutions, achieving digital agility. As a mature integration product since 2005,
+*(branded as WSO2 ESB at the time)*, it continues to be the most sophisticated and extensible enterprise integration 
+solutions out there in open source.
+
+Actively maintained, with commercial support from WSO2 Inc, WSO2 Enterprise Integrator is widely used in production at 
+companies around the globe starting from startups to fortune 500 companies in the fields of government, healthcare, 
+banking, education, communication, etc.
+
+[Installation](https://docs.wso2.com/display/EI640/Installation+Guide) | 
```

Comment:
Is that possible to make it open _blank? AFAIK its not in markdown. Can you try something?

Reply:
Please see https://github.com/abeykoon/product-ei/tree/modify_readme. It is displayed as "|", which I required.